.prettierrc.js

```diff
@@ -0,0 +1,11 @@
+// prettier.config.js or .prettierrc.js
+module.exports = {
+	trailingComma: 'es5',
+	tabWidth: 4,
+	useTabs: true,
+	semi: true,
+	singleQuote: true,
+	'editor.formatOnSave': true,
+	// printWidth: 80,
+	proseWrap: 'always',
```

Comment:
What's the use of `proseWrap: 'always'`? I've never used this before. I'll also learn something from this 🙂

Reply:
https://prettier.io/docs/en/options.html#prose-wrap

Reply:
Thanks 🙂